membership-attribute-service/app/controllers/HealthCheckController.scala

```diff
@@ -17,7 +17,7 @@ class BoolTest(name: String, exec: () => Boolean) extends Test {
 }
 
 class HealthCheckController extends Results {
-  val zuora = NormalTouchpointComponents.zuoraService
+  lazy val zuora = NormalTouchpointComponents.zuoraService
```

Comment:
This hid some spurious errors for those of us who don't have Zuora credentials, but aren't working on it.